### PR TITLE
Name coordinate transformation matrix similar to OoT

### PR DESCRIPTION
**Pull request by engineer124**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

This PR names the coordinate transformation matrices to be consistent with OoT. 

`projectionMatrix` -> `viewProjectionMtxF`
`mf_187FC` -> `billboardMtxF`

I took a look through the asm of z_play to confirm that the same matrix operations occur when extracting the same matrices from z_view. The code around that point does look like there were some changes, but all the matrix operations on these two matrices remain the same

### CODE REVIEW

**Review by EllipticEllipsis — COMMENTED** [review]
So MM only has the billboard MtxF, no Mtx?

**Comment by engineer124** [comment]
> So MM only has the billboard MtxF, no Mtx?

My suspicion is that `/* 0x1883C */ Mtx* unk_1883C;` is `billboardMtx`, but the way it was handled in the mips2c of `Play_Draw` was different enough from OoT that I held back naming it until we could be more confident. The other matrices were obvious due to their interactions with globalCtx->view

**Comment by EllipticEllipsis** [comment]
Right, seems good to hold off on that for now, then. Or we could poke around in an emulator and see what that Mtx actually does.

File: src/overlays/actors/ovl_En_Elforg/z_en_elforg.c [hunk 1]
```diff
@@ -607,7 +607,7 @@ void EnElforg_Draw(Actor* thisx, GlobalContext* globalCtx) {
             AnimatedMat_Draw(globalCtx, Lib_SegmentedToVirtual(&D_0402C818));
             break;
     }
-    Matrix_InsertMatrix(&globalCtx->mf_187FC, 1);
+    Matrix_InsertMatrix(&globalCtx->billboardMtxF, 1);
```
Comment (AngheloAlf) [inline src/overlays/actors/ovl_En_Elforg/z_en_elforg.c:610]: I know it wasn't you, but could you fix this too? c:
```suggestion
    Matrix_InsertMatrix(&globalCtx->billboardMtxF, MTXMODE_APPLY);
```

Comment (engineer124) [reply src/overlays/actors/ovl_En_Elforg/z_en_elforg.c:610]: I double checked and this is already caught in this PR:
https://github.com/zeldaret/mm/pull/454/files

**Review by AngheloAlf — APPROVED** [review]

**Review by hensldm — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**